devito/passes/iet/definitions.py

```diff
@@ -551,10 +551,10 @@ def process(self, graph):
 
 class DeviceAwareDataManager(DataManager):
 
-    def __init__(self, **kwargs):
-        self.gpu_fit = kwargs['options']['gpu-fit']
-        self.gpu_create = kwargs['options']['gpu-create']
-        self.pmode = kwargs['options'].get('place-transfers')
+    def __init__(self, options=None, **kwargs):
+        self.gpu_fit = options['gpu-fit']
```

Comment:
Won't this fail if `options` is not supplied, since it will try to access a `NoneType` object as if it were a dict? Shouldn't the default be `options={'gpu-fit': False, 'gpu-create': False}` or similar?

Reply:
no they are guaranteed to be there by construction

Reply:
I suppose it's only a kwarg for readability then? In practice it is mandatory (and thus more like a positional arg)?